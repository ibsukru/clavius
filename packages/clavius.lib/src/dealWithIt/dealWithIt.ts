const dealWithIt = (value?: string) => (value ? `${value} (▀̿Ĺ̯▀̿ ̿).` : '(▀̿Ĺ̯▀̿ ̿)')

export default dealWithIt
