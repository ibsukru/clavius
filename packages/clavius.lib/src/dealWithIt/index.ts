import dealWithIt from './dealWithIt'

export default dealWithIt
