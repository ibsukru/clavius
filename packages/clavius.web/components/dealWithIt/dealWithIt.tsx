import React from 'react'
import { dealWithIt } from 'clavius.lib'

const DealWithIt: React.FunctionComponent<{ text: string }> = ({ text }) => (
  <span>{dealWithIt(text)}</span>
)

export default DealWithIt
