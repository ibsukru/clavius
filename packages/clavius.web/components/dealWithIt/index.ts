import DealWithIt from './dealWithIt'

export default DealWithIt
