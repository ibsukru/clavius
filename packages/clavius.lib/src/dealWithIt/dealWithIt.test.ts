import dealWithIt from './dealWithIt'

test.each(['Hello world', undefined])('dealWithIt %s', text => {
  expect(dealWithIt(text)).toMatchSnapshot()
})
