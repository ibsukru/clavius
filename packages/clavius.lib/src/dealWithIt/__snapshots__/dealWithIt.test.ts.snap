// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dealWithIt Hello world 1`] = `"Hello world (▀̿Ĺ̯▀̿ ̿)."`;

exports[`dealWithIt undefined 1`] = `"(▀̿Ĺ̯▀̿ ̿)"`;
